src/components/Gradebook/index.jsx

```diff
@@ -220,13 +231,33 @@ export default class Gradebook extends React.Component {
           return acc;
         }, {});
 
-      const totals = { total: `${this.roundGrade(entry.percent * 100)}/100` };
+      const totals = { Total: `${this.roundGrade(entry.percent * 100)}/100` };
       return Object.assign(results, assignments, totals);
     }),
   };
 
   lmsInstructorDashboardUrl = courseId => `${configuration.LMS_BASE_URL}/courses/${courseId}/instructor`;
 
+  formatHeadings = () => {
+    let headings = [...this.props.headings];
+
+    if (headings.length > 0) {
+      const userInformationHeadingLabel = (
+        <div>
+          <div>Username</div>
+          <div className="font-weight-normal student_key">Student Key</div>
+        </div>
+      );
+
+      headings = headings.map(heading => ({ label: heading, key: heading }));
+
+      // replace username heading label to include additional user data
+      headings[0].label = userInformationHeadingLabel;
```

Comment:
I'd like to think of a better way to do this, as this assumes that the username column will always be at this index but have not had time to look into it further.

Reply:
The username is guaranteed to be the first item in our functions that format the api data to look good for a table